base_geoengine/static/src/js/views/geoengine_widgets.js

```diff
@@ -70,15 +78,14 @@ var FieldGeoEngineEditMap = common.AbstractField.extend(geoengine_common.Geoengi
     },
 
     add_tab_listener: function() {
-        var self = this;
-        $('a[data-toggle="tab"]').on('shown.bs.tab', function(e) {
-            var geo_tab_id = self.$el[0].id;
-            var active_tab = $(e.target.hash);
-            if (active_tab.find('#' + geo_tab_id).length == 1) {
-                  self.render_map();
-                  return;
-            }
-        });
+        tab_href = $('#' + this.name).closest('div[role="tabpanel"]')
```

Comment:
Indeed I should better use `this.$el`

However, I can't attach the trigger directly to the pane content. The event "shown.bs.tab" happen on the tab label button.